tests/comment.test.js

```diff
@@ -0,0 +1,112 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../server/app';
+
+chai.use(chaiHttp);
+const baseUrl = '/api/v1/comment';
+
+const comment = {
+  user_id: '57c515a1-890d-412f-8ca1-0a5395123dca',
+  article_id: '7139d3af-b8b4-44f6-a49f-9305791700f4',
+  body: 'A good comment always refereshes the mind',
+};
+
+describe('POST COMMENT', () => {
+  it('should return 400 with invalid or empty payload(user_id)', done => {
+    chai
+      .request(app)
+      .post(baseUrl)
+      .send({
+        article_id: comment.article_id,
+        body: comment.body,
+      })
+      .end((req, res) => {
+        const { status, errors } = res.body;
+        expect(status).to.be.equal(400);
+        expect(res).to.be.an('object');
+        expect(errors.body[0]).to.equal('user_id is required');
+        done();
+      });
+  });
+
+  it('should return 400 with invalid or empty payload(article_id)', done => {
+    chai
+      .request(app)
+      .post(baseUrl)
+      .send({
+        user_id: comment.user_id,
+        body: comment.body,
+      })
+      .end((req, res) => {
+        const { status, errors } = res.body;
+        expect(status).to.be.equal(400);
+        expect(errors.body[0]).to.equal('article_id is required');
+        done();
+      });
+  });
+
+  it('should return 400 with invalid or empty payload(body)', done => {
+    chai
+      .request(app)
+      .post(baseUrl)
+      .send({
+        user_id: comment.user_id,
+        article_id: comment.article_id,
+      })
+      .end((req, res) => {
+        const { status, errors } = res.body;
+        expect(status).to.be.equal(400);
+        expect(errors.body[0]).to.equal('body is required');
+        done();
+      });
+  });
+
+  // it('should return 404 error if article does not exist', done => {
+  //   chai
+  //     .request(app)
+  //     .post(baseUrl)
+  //     .send({
+  //       user_id: comment.user_id,
+  //       article_id: comment.article_id,
+  //       body: comment.body,
+  //     })
+  //     .end((req, res) => {
+  //       const { status, errors } = res.body;
+  //       expect(status).to.be.equal(404);
+  //       expect(errors.body[0]).to.equal('Article does not exist');
+  //       done();
+  //     });
+  // });
```

Comment:
Why do we have this commented out test? @McHardex

Reply:
Oh, when I started work on this branch, the function to be called when that part of the test runs wasn't available yet in our development branch. I will uncomment it and push again.